src/test/globalSetup.test.ts

```diff
@@ -68,19 +68,26 @@ export function getTestLogger(): TestLogger {
     assert.strictEqual(logger, testLogger, 'The expected test logger is not the current logger')
     assert.ok(testLogger, 'TestLogger was expected to exist')
 
-    return testLogger!
+    return logger!
```

Comment:
At least the docstring is wrong now, `getTestLogger()` doesn't provide `testLogger`. Also seems like we need to revisit the existence of `testLogger` entirely.

Reply:
`getTestLogger` isn't really worth anything at this point but the `TestLogger` itself still has value (the setup step sets all log channels to a `TestLogger`) since it lets us check for log entries/levels. I'll see what I can trim.

Reply:
Correction, `getTestLogger` is used to check logged values. In favor of leaving this around for now.